terra/models.py

```diff
@@ -211,9 +219,11 @@ def __repr__(self):
 
 
 class Approval(models.Model):
-    timestamp = models.DateTimeField(auto_now_add=True)
-    approver = models.ForeignKey("Employee", on_delete=models.PROTECT)
-    treq = models.ForeignKey("TravelRequest", on_delete=models.CASCADE)
+    approved_on = models.DateTimeField(auto_now_add=True)
+    approved_by = models.ForeignKey("Employee", on_delete=models.PROTECT)
+    treq = models.ForeignKey("TravelRequest", on_delete=models.PROTECT)
+    fund = models.ForeignKey("Fund", on_delete=models.PROTECT)
+    amount = models.DecimalField(max_digits=10, decimal_places=5)
     type = models.CharField(max_length=1, choices=APPROVAL_TYPES)
```

Comment:
This table no longer needs a `type` field as this table is only about Funding approvals.

Reply:
Removed type, and the APPROVAL_TYPES list.